scripts/ask_query.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+import chromadb
+from fastapi import APIRouter
+from openai import OpenAI
+
+
+openai_client = OpenAI(api_key=os.getenv("OPENAI_API_KEY"))
+chromadb_client = chromadb.HttpClient(
+    host=os.getenv("CHROMADB_HOST", "chromadb"),
+    port=8000,
+    ssl=False,
+    headers=None,
+    settings=chromadb.config.Settings(),
+    tenant=chromadb.config.DEFAULT_TENANT,
+    database=chromadb.config.DEFAULT_DATABASE,
+)
+
+try:
+    learn_collection = chromadb_client.get_collection(name="learn")
+except ValueError:
+    print("There is no collection. I will make one for you!")
+    learn_collection = chromadb_client.create_collection(name="learn")
+
+
+router = APIRouter()
```

Comment:
Let's remove irrelevant operations. In this script, we don't create any API endpoints with FastAPI